### PR TITLE
Update the meta tag

### PR DESCRIPTION
**Pull request by jonkiky**
Update the meta tag
Update the robots.txt to allow indexing all page.

### CODE REVIEW

File: apps/platform/index.html [hunk 1]
```diff
@@ -7,14 +7,17 @@
...
     <meta name="theme-color" content="#FFF" />
     <!-- Open Graph tags -->
     <meta property="og:title" content="Molecular Targets Platform" />
     <meta property="og:type" content="website" />
     <meta property="og:url" content="https://moleculartargets.ccdi.cancer.gov/" />
     <meta
       property="og:description"
-      content="The Molecular Targets Platform is a National Cancer Institute (NCI) instance of the Open Targets Platform with a focus on pediatric cancer data."
+      content="The Molecular Targets Platform is an NCI-supported instance of the Open Targets Platformoutbounnd web site icon with a focus on preclinical pediatric oncology data. It is a tool that supports the identification and prioritization of molecular targets expressed in childhood cancers."
```
Comment (zdorman) [inline apps/platform/index.html]: Are the "outbound web site icon" in lines 10 and 20 here purposeful? Looks like that was a copy/paste leftover.

**Review by zdorman — COMMENTED** [review]
New meta keywords look good.

File: apps/platform/index.html (hunk 1 above)
Comment (jonkiky) [reply apps/platform/index.html]: My bad. Good catch.

**Comment by zdorman** [comment]
All looks good to me after update

**Pull request merged**